### ISSUE TITLE
Calls to setStyle() fail after switching to new (beta) Standard style.

### ISSUE THREAD

**Issue by mdhillman** [mapbox/mapbox-gl-js#12855]
**mapbox-gl-js version**: v3.0.0-beta.1

**browser**: Google Chrome

### Steps to Trigger Behavior

 1. Use new (beta) Standard style
 2. Call setStyle() on the map object using an existing default style (e.g. light-v11).

### Link to Demonstration

https://jsfiddle.net/6dxvfns1/2/

### Expected Behavior

Map updates to pull visual style from the input style URL.

### Actual Behavior

An AJAX error occurs during the fetch request.

**Comment by mdhillman** [comment]
Note that using the setStyle() function to switch __to__ the new (beta) Standard style works as expected, issue only occurs once the Standard style is applied.

**Comment by TimBarham** [comment]
As a workaround, I've found if I explicitly specify the new Standard style when I create the map (rather than not setting a style and it defaulting to the Standard style), I can switch between the standard style and older styles (and back again) without issue...

``` ts
const map: mapboxgl.Map = new mapboxgl.Map({
    container: "map",
    style: "mapbox://styles/mapbox/standard-beta",
    center: [lng, lat],
    zoom,
});
```

**Comment by stepankuzmin** [comment]
Thanks, everyone! This issue will be fixed in the upcoming GL JS v3.0.0-beta.2 release
